server/services/healthCheck.ts

```diff
@@ -37,9 +37,10 @@ function addAppInfo(result: HealthCheckResult): HealthCheckResult {
 
 function getBuild() {
   try {
-    // eslint-disable-next-line import/no-unresolved,global-require
+    // eslint-disable-next-line import/no-unresolved,global-require, @typescript-eslint/no-require-imports
     return require('../../build-info.json')
-  } catch (ex) {
+    // eslint-disable-next-line  @typescript-eslint/no-unused-vars
```

Comment:
it is required to leave //eslint-dsiable , i refactor ex to _ex to follow patterns for this non used vars